packages/node/test/stubs.d.ts

```diff
@@ -4,6 +4,8 @@
 declare module 'lavamoat-core/test/util.js' {
   export interface PlatformRunScenarioOpts {
     // NOTE: this loads scenario.d.ts!
+    // ALSO: it cannot be replaced with an import statement because reasons
+    // eslint-disable-next-line @typescript-eslint/consistent-type-imports
```

Comment:
(since TS doesn't support `import type()` inline, only top-level)

Reply:
specifically, this is what doesn't want to work:

```ts
import type {NormalizedScenario} from 'lavamoat-core/test/scenarios/scenario.js'

declare module 'lavamoat-core/test/util.js' {
// ...
}
```